workflows/prognostic_c48_run/tests/test_fv3fit_transformer.py

```diff
@@ -57,12 +58,18 @@ def add_one_to_temperature():
         state["air_temperature"] += 1
         return {"some_diag": state["specific_humidity"]}
 
+    temperature_before = state["air_temperature"].copy(deep=True)
     out = transform(add_one_to_temperature)()
+    temperature_after = state["air_temperature"].copy(deep=True)
 
     # ensure tendency of internal energy is non-zero somewhere (GH#1433)
     max = abs(out["tendency_of_internal_energy_due_to_machine_learning"]).max()
     assert max.values.item() > 1e-6
 
+    np.testing.assert_allclose(
+        temperature_after, temperature_before + 900 / 86400, rtol=1e-5
+    )
+
     # sort to make the check deterministic
     regression_state(out, regtest)
```

Comment:
Is there any value to the regression test anymore?

Reply:
There are other features that are captured by the regression test, like monitoring of variables and output of diagnostics.